pkg/metainfo/kvmetainfo/buckets_test.go

```diff
@@ -274,14 +275,14 @@ func runTest(t *testing.T, test func(*testing.T, context.Context, *testplanet.Pl
 	testplanet.Run(t, testplanet.Config{
 		SatelliteCount: 1, StorageNodeCount: 4, UplinkCount: 1,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
-		db, streams, err := newMetainfoParts(planet)
+		db, streams, err := newMetainfoParts(planet, t)
 		require.NoError(t, err)
 
 		test(t, ctx, planet, db, streams)
 	})
 }
 
-func newMetainfoParts(planet *testplanet.Planet) (*kvmetainfo.DB, streams.Store, error) {
+func newMetainfoParts(planet *testplanet.Planet, t *testing.T) (*kvmetainfo.DB, streams.Store, error) {
```

Comment:
Make `t` the first argument rather than last, usually `testing.T` is the first argument.